### PR TITLE
multibyte message was broken

### PR DESCRIPTION
**Pull request by nojimage**
I write an acceptance test as below,

```
<?php
$I = new AcceptanceTester($scenario);
$I->wantTo('日本語のメッセージを書きたい');
```

I run this testcase, results expect that
```
Acceptance Tests (1) -------------------------------------------------------------------------------------------------
日本語のメッセージを書きたい (MultibyteCept)                                                      Ok
----------------------------------------------------------------------------------------------------------------------
```

but, actual get
```
Acceptance Tests (1) -------------------------------------------------------------------------------------------------
Ɨ�本語のメッセージを書きたい (MultibyteCept)                                                      Ok
----------------------------------------------------------------------------------------------------------------------
```

The reason for this problem is using `strtoupper` at output messages.

The `strtoupper` will be broken multibyte string.
I think it is better that you use instead of the `mb_strtoupper`.

### CODE REVIEW

**Comment by nojimage** [comment]
Please merge PR, if there is no problem.

File: src/Codeception/Lib/Console/Message.php [hunk 1]
```diff
@@ -32,7 +33,7 @@ public function style($name)
 
     public function width($length, $char = ' ')
     {
-        $message_length = strlen(strip_tags($this->message));
+        $message_length = Multibyte::strwidth(strip_tags($this->message));
```
Comment (samdark) [inline src/Codeception/Lib/Console/Message.php]: Better to name it strlen still.

File: src/Codeception/Util/Multibyte.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+namespace Codeception\Util;
+
+/**
+ * Multibyte handling methods
+ */
+class Multibyte
+{
+
+    /**
+     * strlen
```
Comment (samdark) [inline src/Codeception/Util/Multibyte.php]: Should be properly described.

File: src/Codeception/Util/Multibyte.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    }
+
+    /**
+     * strwidth
+     *
+     * @param string $string
+     * @param string $encoding
+     * @return string
+     */
+    public static function strwidth($string, $encoding = 'UTF-8')
```
Comment (samdark) [inline src/Codeception/Util/Multibyte.php]: Why not strlen?

File: src/Codeception/Util/Multibyte.php [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+     * @param string  $string
+     * @param integer $start
+     * @param integer $length
+     * @param string  $encoding
+     * @return string
+     */
+    public static function substr($string, $start, $length = null, $encoding = 'UTF-8')
+    {
+        if (!function_exists('mb_substr')) {
+            $length = is_null($length) ? strlen($length) : $length;
```
Comment (samdark) [inline src/Codeception/Util/Multibyte.php]: `is_null` should be replaced with `$length === null`

Comment (sergeyklay) [inline src/Codeception/Util/Multibyte.php]: why?

**Comment by kkopachev** [comment]
What's the point in introducing new class with mb_functions existence check, if in composer.json ext-mbstring is required anyway, so all these functions are available? Plus test will fail if there is no mbstring extension installed.

File: src/Codeception/Lib/Console/Message.php (hunk 1 above)
Comment (nojimage) [inline src/Codeception/Lib/Console/Message.php]: mb_strwidth is return width of string. multibyte characters are usually twice the width of single byte characters.

**Comment by nojimage** [comment]
@kkopachev I had overlooked ext-mbstring required in composer.json.  :p
The Multibyte class does not need.

File: src/Codeception/Util/Multibyte.php (hunk 4 above)
Comment (samdark) [inline src/Codeception/Util/Multibyte.php]: There's no need to use function in this case.

Comment (sergeyklay) [inline src/Codeception/Util/Multibyte.php]: `is_null($length) ? strlen($length)` :smiley_cat: I just noticed on this design

**Comment by DavertMik** [comment]
this PR can't be merged automatically. Please resend it

**Pull request closed without merging**